dspace-api/src/main/java/org/dspace/fileaccess/FileAccessFromMetadataServiceImpl.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.fileaccess;
+
+import java.sql.*;
+import javax.servlet.http.*;
+import org.dspace.authorize.*;
+import org.dspace.content.*;
+import org.dspace.core.*;
+import org.dspace.fileaccess.service.*;
+import org.dspace.importer.external.elsevier.entitlement.*;
+
```

Comment:
I think I have a rough idea why you want to add this class. But could you please add a class comment why such a stub is added to DSpace?